src/Evolution/DiscontinuousGalerkin/Actions/ComputeTimeDerivative.hpp

```diff
@@ -353,7 +354,9 @@ struct ComputeTimeDerivative {
   using const_global_cache_tags = tmpl::append<
       tmpl::list<::dg::Tags::Formulation,
                  evolution::Tags::BoundaryCorrection<EvolutionSystem>,
-                 domain::Tags::ExternalBoundaryConditions<Dim>>>;
+                 domain::Tags::ExternalBoundaryConditions<Dim>>,
+      tmpl::conditional_t<LocalTimeStepping,
+                          tmpl::list<::Tags::MinimumTimeStep>, tmpl::list<>>>;
```

Comment:
Why add this is more than one place? Once in a LTS action seems like it should be enough

Reply:
This one is for LTS, the one in `src/Time/ChangeSlabSize/Action.hpp` is for GTS (which will usually also be used for LTS), and the one in `src/Time/Actions/ChangeStepSize.hpp` is for CCE.